src/GafferSceneUI/StandardLightVisualiser.cpp

```diff
@@ -763,9 +772,6 @@ IECoreGL::ConstRenderablePtr StandardLightVisualiser::distantRays()
 IECoreGL::ConstRenderablePtr StandardLightVisualiser::spotlightCone( float innerAngle, float outerAngle, float lensRadius, float length, float lineWidthScale )
 {
 	IECoreGL::GroupPtr group = new IECoreGL::Group();
-	addWireframeCurveState( group.get() );
-	addConstantShader( group.get(), false );
```

Comment:
This is changing the behaviour of a public method, such that visualisations produced with it won't have a bunch of the necessary state.